examples/periodic_tasks/src/main.rs

```diff
@@ -108,6 +113,7 @@ fn main() -> ! {
         Clic::ie(Interrupt::Timer1Cmp).set_pcs(true);
         Clic::ie(Interrupt::Timer2Cmp).set_pcs(true);
         Clic::ie(Interrupt::Timer3Cmp).set_pcs(true);
+        //Clic::ie(Interrupt::MachineTimer).set_pcs(true);
```

Comment:
Oh, this makes a lot of sense :D

Anyway. Commented out code is discouraged. Please remove the line. It's already apparent from the diff that "this line was removed", and having it be commented out does not add value.